lib/runtime/wasmer/instance.go

```diff
@@ -26,15 +26,17 @@ import (
 // Name represents the name of the interpreter
 const Name = "wasmer"
 
-// Check that runtime interfaces are satisfied
 var (
+	// Check that runtime interfaces are satisfied
 	_ runtime.Instance = (*Instance)(nil)
 	_ runtime.Memory   = (*wasm.Memory)(nil)
 
 	logger = log.NewFromGlobal(
 		log.AddContext("pkg", "runtime"),
 		log.AddContext("module", "go-wasmer"),
 	)
+
+	substrateWasmCompressionFlag = []byte{82, 188, 83, 118, 70, 219, 142, 5}
```

Comment:
FYI to have constant byte slices one could have it as a constant string which then gets converted to a byte slice where it's needed.